tests/unit/test_routes/test_resource_retreival.py

```diff
@@ -144,5 +148,27 @@ def test_filters(module_client, module_db):
     for resource in response.json['data']:
         assert (resource.get('category') == "Back End Dev")
 
-    # TODO: Filter by updated_after
-    # (Need to figure out how to manually set last_updated and created_at)
+    # Filter by updated_after
```

Comment:
I think that `test_filters` is testing too many things in one function. Let's break this up into `test_filter_languages`, `test_filter_category` and `test_filter_updated_after`

Reply:
I think so too, we already separated the paid filter, might as well separate those too. Also I just realized that we didn't include the case that there might not be any resources with said languages, perhaps giving a gibberish language and asserting it returns 404? I might be wrong so I would love to know more.

Reply:
Good call, we should add that test. Same with `/api/v1/resources?category=gibberish` returning a 404